classes/tdx-signed-fit-image.inc

```diff
@@ -8,6 +8,7 @@ UBOOT_SIGN_ENABLE:verdin-am62-k3r5 = "0"
 UBOOT_MKIMAGE_DTCOPTS = "-I dts -O dtb -p 2000"
 UBOOT_SIGN_KEYDIR ?= "${TOPDIR}/keys/fit"
 UBOOT_SIGN_KEYNAME ?= "dev"
+UBOOT_SIGN_IMG_KEYNAME ?= "dev2"
```

Comment:
@rborn-tx thanks for reviewing!

> @sergioprado I guess this will impact i.MX users as well, right? If an i.MX user has set FIT_GENERATE_KEYS=0 then I suppose this new key will cause the build to fail and they'll need to create the "dev2" key as well, no?

You are right. But I feel this is more of a corner case, then the default. We might have a few users, and I would say most of them (if not all) are probably using the default configuration (`FIT_GENERATE_KEYS=1`). In my opinion, this breaking change is not worth documenting, since it might leave the documentation more confusing. I doubt if there is a user in this situation, but if there is, I would prefer to receive a bug report and provide some instruction on how to fix it than to pollute the documentation.

Does that make sense? Please let me know if you see any issue with this approach.

Reply:
> But in the default case, the "dev2" key would be generated and used for signing the kernel image without user knowledge. If they were doing this in CI, for example, it's likely that this new key would be lost forever. Then, each image produced in CI would have a different "dev2" key which, I assume, would prevent OS updates in the field afterwards. Or am I missing something?

Yes, you are right in your assumption. The only thing you are missing is that this key is currently not being used because we have `FIT_SIGN_INDIVIDUAL="0"` in our default configuration. That means our current implementation is only signing the configuration node from the FIT image, not the individual images.

Reply:
Just hopping on this thread since my question is related. This change with `UBOOT_SIGN_IMG_KEYNAME` is independent/separate to the AM62 boot artifacts signing, correct? Or is their a relation here I'm not understanding?

Reply:
> Just hopping on this thread since my question is related. This change with UBOOT_SIGN_IMG_KEYNAME is independent/separate to the AM62 boot artifacts signing, correct? Or is their a relation here I'm not understanding?

Exactly. This is a small issue I found in the FIT image implementation. Totally unrelated to the AM62 implementation. This is why this change is in a separated commit.